Code/training_team8.py

```diff
@@ -24,8 +25,8 @@
 # %% ----------------------------------- Hyper Parameters --------------------------------------------------------------
 LR = 5e-3
 MOMENT = .9
-BATCH_SIZE = 8
-N_EPOCHS = 50
+BATCH_SIZE = 16
+N_EPOCHS = 15
```

Comment:
so far. on the full training set. I'm getting ~36 minutes per epoch. 
so 15 * 30 ~= 7.5 hours. If i'm right, I'll be done around 2/3am